docs/rsdl/rapid-pro-rsdl-intro.md

```diff
@@ -3,80 +3,238 @@ id: rsdl-intro
 title: RAPID SDL intro
 ---
 
-# Introduction to RAPID Pro schema definition language (RSDL)
+# Introduction to RAPID Schema Definition Language (RSDL)
 
-> DRAFT
+RAPID Schema Definition Language (RSDL) is language to describe Web APIs.
 
-RAPID Pro schema definition language (RSDL) is language to describe Web APIs.
-RSDL is based on a [profile](<https://en.wikipedia.org/wiki/Profile_(engineering)>) of the
-[OData](https://en.wikipedia.org/wiki/Open_Data_Protocol) specification with the goal to provide an easy way
-to create a Web API that is compatible with OData and can evolve into a more elaborate version.
+RSDL is based on the [RAPID PROfile](<https://en.wikipedia.org/wiki/Profile_(engineering)>) of the
+[OData](https://en.wikipedia.org/wiki/Open_Data_Protocol) specification. RAPID provides an easy way
+to envision, create, and consume a Web API that is compatible with the OData Standard and can evolve over time to support more advanced scenarios.
 
 ## Introductory Example
 
-The core description of the "shape" of the API is given by a RSDL document.
+RAPID APIs are defined by a schema, which can easily be described through RSDL.
 
-Below is a first example of an RSDL file that describes an API of a service that provides data on companies and their employees
+### Defining a Type
+Let's say that we wanted our API to deal with information about a company. We could describe the properties of a company as follows:
 
-```
+~~~rsdl
 type company
```

Comment:
```suggestion
type Company
```

Use capital initial letters for types? Is a typical convention: types start with capital, properties/fields start with lowercase. Also makes `company: Company` more digestible

Reply:
I played with this a bit, and after I made the change then the names of types were the **only** place in the entire syntax that would use initial caps (keywords, attribute names, property names, etc. are all lower camel case), which made type names stand out and look a bit strange to me.

Reply:
I note that GraphQL does use capitals for type names; both custom names as well as built-in types, so it looks a little more consistent.

If we think the conventions should be initial caps for custom type names we should do the same for built-in primitive types.

Reply:
I am in favor of using initial caps for all types and initial lower case for all structural features (i.e, properties, terms, attributes, methods, ... )

Reply:
I'm used to initial caps for type names, it is widely used for class names, e.g. in Java, JavaScript, Typescript, only a convention and not enforced by the "compiler". And it is also widely used in entity-relationship modeling.